graphql/models/City.graphql

```diff
@@ -0,0 +1,24 @@
+type Query{
+    cities:[City!]! @paginate(defaultCount:10)
+    city(id:ID @eq):City @find
+}
+
+type City{
+    id: ID!
+    name : String!
+    area : String!
+    gover_id : String!
+    is_division : String!
+    districts : String!
+    divisions : String!
+    units : String!
+    population : String!
+    p_density_rate : String!
+    creation_date : String!
+    national_day : String!
```

Comment:
could we use camelCase for the GraphQL models and responses so instead of `national_day` it should be `nationalDay`